build.gradle

```diff
@@ -73,7 +73,7 @@ task quickBuild {
     dependsOn ':Corona-Warn-App:assembleDeviceRelease'
     dependsOn ':Corona-Warn-App:testDeviceReleaseUnitTest'
     dependsOn ':Corona-Warn-App:lintDeviceRelease'
-    dependsOn ':Corona-Warn-App:ktlintDeviceReleaseCheck'
+    dependsOn ':Corona-Warn-App:ktlintDeviceReleaseSourceSetCheck'
```

Comment:
can you please change also in `quickBuildWithFixes` task ?

Reply:
we could even think about removing the quick from the name :D
![image](https://user-images.githubusercontent.com/64483219/125461789-eafa50ed-ea6b-4f42-8162-e2fef9656ff5.png)

Reply:
> we could even think about removing the quick from the name :D
You're lucky, it takes me 28m on average...

Reply:
> can you please change also in `quickBuildWithFixes` task ?

Done